test/CDFDocumentTest.php

```diff
@@ -145,7 +145,7 @@ public function testMergeDocumentsByKeys($setOne, $setTwo)
    */
   public function testMergeDocumentsNoOverlap($setOne, $setTwo, $elementFromSetTwo)
   {
-    $this->cdfDocument->setCDFEntities(...$setOne);
+    $this->cdfDocument->setCdfEntities(...$setOne);
     $documentToMerge = new CDFDocument(...$setTwo);
```

Comment:
And CDFDocument?

Reply:
Out of scope. Please see my overall comment on this PR.